components/recruitment/index.tsx

```diff
@@ -108,36 +176,55 @@ const DEPARTMENTROLES = [
           </div>
         </div>
 */}
-      </div>
+        </div>
 
         <div className="section-my pb-0">
-          <h2 className="font-bold text-center text-2xl text-[#002750] mb-5 sm:text-4xl sm:mb-16">
+          <h2 className="mb-5 text-center text-2xl font-bold text-[#002750] sm:mb-16 sm:text-4xl">
             NUS Fintech Society Recruitment for AY 2023/2024
           </h2>
 
           <p className=" text-left text-sm text-[#002750] sm:text-xl">
-            At NUS Fintech Society, we work on <strong>industry-grade</strong> financial digital
-            transformation projects, carry out research projects and educate
-            students with Fintech knowledge. From blockchain developer to
-            software engineer to project management executive, we have <strong>both
-            technical and operational roles</strong>  for all who would like to join us.
+            At NUS Fintech Society, we work on <strong>industry-grade</strong>{' '}
+            financial digital transformation projects, carry out research
+            projects and educate students with Fintech knowledge. From
+            blockchain developer to software engineer to project management
+            executive, we have{' '}
+            <strong>both technical and operational roles</strong> for all who
+            would like to join us.
           </p>
 
-          <h2 className="font-bold text-left text-2xl text-[#002750] my-8 sm:text-4xl sm:my-16">
+          <h2 className="my-8 text-left text-2xl font-bold text-[#002750] sm:my-16 sm:text-4xl">
             Who Should Join Us?
           </h2>
 
           <p className=" text-left text-sm text-[#002750] sm:text-xl">
-            The NUS FinTech Society is looking for <strong> passionate and driven people </strong> 
+            The NUS FinTech Society is looking for{' '}
+            <strong> passionate and driven people </strong>
             looking to be at the forefront of cutting edge technology in the
-            FinTech space. Learners of <strong>all skill levels are welcome</strong> and interested
-            parties can discover more from the <strong>roles listed below.</strong>
+            FinTech space. Learners of{' '}
+            <strong>all skill levels are welcome</strong> and interested parties
+            can discover more from the <strong>roles listed below.</strong>
           </p>
-
         </div>
       </MaxWidth>
+      {isTechnical
+        ? TECHNICAL.map((department: DepartmentRoleProps, index) => (
+            <DepartmentRoles
+              key={index}
+              department={department.department}
+              roles={department.roles}
+            />
+          ))
+        : OPERATIONS.map((department: DepartmentRoleProps, index) => (
+            <DepartmentRoles
+              key={index}
+              department={department.department}
+              roles={department.roles}
+            />
+          ))}
+      <SignUp />
```

Comment:
Could we have green for operations wing `SectionHeader`s? Since the `RoleCard`s should also be green themed

Reply:
Done!